SDWebImage/SDWebImageDownloaderOperation.h

```diff
@@ -41,6 +41,9 @@ FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadFinishNotification
 - (nullable NSURLCredential *)credential;
 - (void)setCredential:(nullable NSURLCredential *)value;
 
+- (nullable NSURLRequest *)request;
```

Comment:
This API actually should mark as open, since this is what a `download operation` basic information. And we have a `SDWebImageDownloadStartNotification` notification, which will contains a `NSOperation<SDWebImageDownloadOperationInterface`. User can use the `request` to identifier which operation to check, use `response` to get the information